tests/benchmarks/cli/stories/test_partial_remove.py

```diff
@@ -0,0 +1,20 @@
+import glob
+import os
+import random
+
+
+def test_partial_remove(bench_dvc, tmp_dir, dvc, dataset, remote):
+    # Add/push full dataset
+    dvc.add(str(dataset))
+    dvc.push()
+
+    # Remove some files
+    for f in glob.glob("*", root_dir=dataset, recursive=True):
+        if random.random() > 0.5:
+            os.remove(dataset / f)
+
+    # Benchmark operations for removing files from dataset
+    bench_dvc("add", dataset)
+    bench_dvc("push")
+    bench_dvc("gc", "-f", "-w")
+    bench_dvc("gc", "-f", "-w", "-c", name="cloud")
```

Comment:
gc test seems to be the same for `partial add` and `partial remove` tests, right? Do you think these should be just one test?

Reply:
I don't think so. `gc` after partial add should be a noop since there's nothing deleted (except maybe a .dir file), so it's a different scenario than the partial add `gc`, which actually deletes files from the cache/remote.